src/storm-pars-cli/storm-pars.cpp

```diff
@@ -1,3 +1,5 @@
+#include <_types/_uint64_t.h>
```

Comment:
```suggestion
#include <cstdint.h>
```

`_types/_uint64_t.h` is not a standard header, right?

Reply:
My autocomplete keeps doing this, thanks for the catch